src/daemon/dlt-daemon.c

```diff
@@ -835,6 +855,171 @@ int option_file_parser(DltDaemonLocal *daemon_local)
     return 0;
 }
 
+#ifdef DLT_LOG_LEVEL_APP_CONFIG
+/**
+ * Load configuration file parser
+ */
+
+static int compare_app_id_conf(const void *lhs, const void *rhs)
+{
+    return strncmp(((DltDaemonContextLogSettings *)lhs)->apid,
+                   ((DltDaemonContextLogSettings *)rhs)->apid, DLT_ID_SIZE);
+}
+
+int app_id_default_log_level_config_parser(DltDaemon *daemon,
+                                           DltDaemonLocal *daemon_local) {
+    FILE *pFile;
+    char line[value_length - 1];
+    char app_id_value[value_length];
+    char ctx_id_value[value_length];
+    DltLogLevelType log_level_value;
+
+    char *pch;
+    const char *filename;
+
+    /* open configuration file */
+    filename = daemon_local->flags.avalue[0]
+                   ? daemon_local->flags.avalue
+                   : CONFIGURATION_FILES_DIR "/dlt-log-levels.conf";
+
+    pFile = fopen(filename, "r");
+    if (pFile == NULL) {
+        dlt_vlog(LOG_WARNING, "Cannot open app log level configuration%s\n", filename);
+        return -errno;
+    }
+
+    /* fetch lines from configuration file */
+    while (fgets(line, value_length - 1, pFile) != NULL) {
+        pch = strtok(line, " ");
```

Comment:
Please add a tab to the delimiters:
strtok(line, " \t");

Reply:
added tab additionally, so both are possible